cedar-policy-core/src/est/expr.rs

```diff
@@ -914,35 +915,17 @@ impl TryFrom<&Node<Option<cst::Relation>>> for Expr {
             }
             cst::Relation::Has { target, field } => {
                 let target_expr = target.try_into()?;
-                let mut errs = vec![];
-                if let Some(field_expr) = field.to_expr_or_special(&mut errs) {
-                    if let Some(attr) = field_expr.into_valid_attr(&mut errs) {
-                        return Ok(Expr::has_attr(target_expr, attr));
-                    }
-                }
-                if errs.is_empty() {
-                    Err(field
-                        .to_ast_err(ToASTErrorKind::InvalidAttribute(
-                            field.try_as_inner()?.to_smolstr(),
-                        ))
-                        .into())
-                } else {
-                    Err(errs.into())
-                }
+                field
+                    .to_expr_or_special()?
+                    .into_valid_attr()
+                    .map(|attr| Expr::has_attr(target_expr, attr))
```

Comment:
Yes, it's used in the CST->AST translation of attributes. I suspect that in the previous code, it would never be the case that `errs.is_empty()`. In case of an error, `errs` would probably just contain `InvalidAttribute`.